civiclens/nlp/comments.py

```diff
@@ -3,8 +3,8 @@
 from networkx.algorithms.community import louvain_communities
 from sentence_transformers import SentenceTransformer, util
 
-from ..utils.database_access import Database, pull_data
-from .tools import RepComments
+from civiclens.nlp.tools import RepComments
```

Comment:
@michplunkett I'm finding I'm having to write out the full path to get any of my scripts to run. Am I wrong in thinking that relative imports (`from .tools import RepComments`) should suffice instead, or is there something else wrong here?

Reply:
Personally I was only finding this issue when running the code on the linux server, not on my local machine. Can't think of why that would change the structure of relative imports though